### PR TITLE
WEBAPP-346: Enforce flow annotation on all files

### PR DESCRIPTION
**Pull request by Schedulaar**
This pull request belongs to an issue on our [bugtracker](https://integreat.atlassian.net/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

**Comment by Schedulaar** [comment]
We need to exclude eslint for /tools/*.js, because we don't use it there. Should we create a separate eslint config for node scripts?

**Comment by Schedulaar** [comment]
Nvm, I've added another .eslintrc in src folder, so that we have the only the basic .eslintrc for the tools folder.

**Comment by Schedulaar** [comment]
We should highly consider creating an own flow preset right away :D

File: src/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "@integreat-app/eslint-config",
+    "@integreat-app/eslint-config-react",
+    "@integreat-app/eslint-config-jest",
+    "plugin:flowtype/recommended"
+  ],
+  "plugins": [
+    "flowtype"
+  ],
+  "rules": {
+    "flowtype/require-valid-file-annotation": [
```
Comment (maxammann) [inline src/.eslintrc.json]: These rules should be in @integreat-app/eslint-config

Comment (Schedulaar) [reply src/.eslintrc.json]: Sure, these rules, shouldn't be right here.
Take a look at https://issues.integreat-app.de/browse/LINT-4
Do you agree with that?

**Pull request merged**